langkit/ocaml_api.py

```diff
@@ -369,6 +369,29 @@ def is_struct(self, type, from_module=None):
             (T.EnvRebindings, lambda _: False),
         ])
 
+    def get_finalize(self, type):
+        """
+        Return the name of the finalization function if a value of the given
+        type needs a finalization. A value needs a finalization if it is
+        completly converted to an OCaml function. Thus unwrapping
```

Comment:
Sorry for this newbie question :) Are the values for `type` really converted to functions? (that sounds weird: how does wrapping/unwrapping work on functions?)

Reply:
Not a newbie question at all, that was a typo. We create OCaml values and not functions

Reply:
Ok, thanks!